README.md

```diff
@@ -1,12 +1,27 @@
 # Bangazon Web Initial Site
 
+## Dependencies
+
+To ensure that the Initial Bangazon App works as intended make sure that you have the following dependencies and technologies on your local machine
+
+1. dotnet 
+
+If you need to download dotnet on to your local machine, please visit [Microsoft's Documentation](https://www.microsoft.com/en-us/download/details.aspx?id=30653)
+
+2. bower
+
+If you need to download bower on to your local machine, please visit [Bower's Documentation](https://bower.io/)
+
 ## Installation
 
 Clone or fork the project. Navigate to where the project is saved on your machine.
+Please note that if running the application in OSX/UNIX/Windows that the commands below require for you to manually change the path to your database.
```

Comment:
"Please note that if running the application in OSX/UNIX/Windows "

--> "Please note that you must manually change the path to your database before running the commands below"